site/docs/components/stepped-tracker/index.mdx

```diff
@@ -9,19 +9,16 @@ data:
     [
       "Discrete progress indicator",
       "Progress indicator",
-      "Progress stepper",
+      "Progress SteppedTracker",
       "Progress steps",
       "Screen flow",
       "Status tracker",
       "Stepper",
     ]
   relatedComponents:
     [
-      { name: "Breadcrumbs", relationship: "similarTo" },
-      { name: "Pagination", relationship: "similarTo" },
-      { name: "Progress", relationship: "similarTo" },
+      { name: "Accordion", relationship: "contains" },
```

Comment:
This uses accordion?

Reply:
No, messed up the relationship, it should be similarTo

Reply:
Done!